config/jobs/kubernetes/sig-cloud-provider/azure/release-master.yaml

```diff
@@ -527,7 +527,7 @@ periodics:
         make e2e-test
       env:
       - name: KUBERNETES_VERSION
-        value: "latest"
+        value: "latest-master"
```

Comment:
This was autogenerated, looks like generated files are out of date... looking into it

Reply:
fixed in https://github.com/kubernetes/test-infra/pull/26775/commits/5ab5cd2488bd3a3afd9624941d9ac0b2999c01af